.travis.yml

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#
+# Tencent is pleased to support the open source community by making TubeMQ available.
+#
+# Copyright (C) 2012-2019 Tencent. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at
+#
+# https://opensource.org/licenses/Apache-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+sudo: required
+dist: trusty
+language: java
+
+jdk:
+  - oraclejdk8
```

Comment:
not sure if we should use openjdk or oraclejdk here

Reply:
+1 for openjdk, +0 for tencent jdk.

Reply:
+1 for openjdk

Reply:
+1 for openjdk.